x-pack/plugin/ml/qa/single-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/LearningToRankRescorerIT.java

```diff
@@ -242,30 +241,33 @@ public void testLearningToRankRescoreSmallWindow() throws Exception {
                 "learning_to_rank": { "model_id": "ltr-model" }
               }
             }""");
-        assertHitScores(client().performRequest(request), List.of(20.0, 20.0, 1.0, 1.0, 1.0));
```

Comment:
Hmm, I wonder how the PR that added the assertion passed in the first place 🤔 Maybe this was skipped when the functionality changed to start throwing when the window size is too small?

Reply:
Yea, I think that this test was muted when the the small window behaviour was changed.